test/files/neg/warn-unused-params.scala

```diff
@@ -75,28 +77,31 @@ trait Unimplementation {
 }
 
 trait DumbStuff {
-  def f(implicit dummy: DummyImplicit) = 42
-  def g(dummy: DummyImplicit) = 42
+  def f(implicit dummy: DummyImplicit) = answer
+  def g(dummy: DummyImplicit) = answer
 }
 trait Proofs {
-  def f[A, B](implicit ev: A =:= B) = 42
-  def g[A, B](implicit ev: A <:< B) = 42
-  def f2[A, B](ev: A =:= B) = 42
-  def g2[A, B](ev: A <:< B) = 42
+  def f[A, B](implicit ev: A =:= B) = answer
+  def g[A, B](implicit ev: A <:< B) = answer
+  def f2[A, B](ev: A =:= B) = answer
+  def g2[A, B](ev: A <:< B) = answer
 }
 
 trait Anonymous {
-  def f = (i: Int) => 42      // warn
+  def f = (i: Int) => answer      // warn
 
-  def f1 = (_: Int) => 42     // no warn underscore parameter (a fresh name)
+  def f1 = (_: Int) => answer     // no warn underscore parameter (a fresh name)
 
   def f2: Int => Int = _ + 1  // no warn placeholder syntax (a fresh name and synthetic parameter)
 
-  def g = for (i <- List(1)) yield 42    // warn map.(i => 42)
+  def g = for (i <- List(1)) yield answer    // warn map.(i => 42)
 }
 trait Context[A]
 trait Implicits {
-  def f[A](implicit ctx: Context[A]) = 42
-  def g[A: Context] = 42
+  def f[A](implicit ctx: Context[A]) = answer
+  def g[A: Context] = answer
 }
 class Bound[A: Context]
+object Answers {
```

Comment:
this makes it sound like there's more than one 🤔

Reply:
I already suggested that by not ascribing the literal type...

Reply:
I see... So we need to [keep on searching](https://www.youtube.com/watch?v=dTlGZOsvB5Q)

Reply:
@lrytz thanks I didn't know that one